modules/imgproc/perf/perf_warp.cpp

```diff
@@ -42,11 +42,7 @@ PERF_TEST_P( TestWarpAffine, WarpAffine,
 
     TEST_CYCLE() warpAffine( src, dst, warpMat, sz, interType, borderMode, borderColor );
 
-#ifdef __ANDROID__
-    SANITY_CHECK(dst, interType==INTER_LINEAR? 5 : 10);
-#else
     SANITY_CHECK(dst, 1);
```

Comment:
Maybe `SANITY_CHECK_NOTHING()`? We're trying to deprecate accuracy checks in perf tests for some time (https://github.com/opencv/opencv/wiki/HowToWritePerfTests#simple-regression-check).

Reply:
Let's presume the checks, where we have it. No check for new code.